### ISSUE TITLE
Delete Operation

### ISSUE THREAD

**Issue by dlecocq** [igrigorik/bloomfilter-rb#37]
It came up in another repo, but it appears the non-counting implementations in here attempt to support deletions. Perhaps instead of trying to support deletions in those cases, it should instead throw an error? Seems pretty misleading.

Specifically https://github.com/igrigorik/bloomfilter-rb/blob/master/lib/bloomfilter/redis.rb#L47

**Comment by igrigorik** [comment]
Oi, I need a stronger coffee today... To correct my earlier point: false positives are possible _and_ with deletes (without counting logic) all bets are off about the error rate. To make matters worse, as you correctly pointed out, we also open ourselves to false negatives: delete resets a bit previously set by another entry, thus causing that entry to be mis-reported.

tl;dr: it's an implementation bug.

**Comment by igrigorik** [comment]
That should do it! Thanks for the heads up.
